### PR TITLE
Adapt module docstring

### LINKED ISSUE #210: Adapt module docstring
The metamodel documentation is confusing as so far that it is not clear whether the work presented herein is supported by IDTA or is independent.

For example, a disclaimer such as this one is missing:

> The presented version of the Metamodel within this documentation is related to the work of aas-core-works, which can be found here. The presented content of this website is neither related to the IDTA nor Plattform Industrie 4.0 and does not represent an official document. Content of Metamodel with Version v3.0. Any static content or specific content for this model version can be presented here.

### PR DESCRIPTION
**Pull request by s-heppner**
Fixes #210.

### CODE REVIEW

File: aas_core_meta/v3.py [hunk 1]
```diff
@@ -1,5 +1,11 @@
 """
-Provide the meta-model for Asset Administration Shell V3.0.
+The presented version of the Metamodel within this documentation
+is related to the work of aas-core-works, which can be found here.
```
Comment (g1zzm0) [inline aas_core_meta/v3.py:3]: Add Link to aas-core-works

File: aas_core_meta/v3.py [hunk 2]
```diff
@@ -1,5 +1,11 @@
...
+The presented content of this website is neither related to the IDTA nor
```
Comment (g1zzm0) [inline aas_core_meta/v3.py:4]: The presented content of this documentation is no official release of the IDTA or

File: aas_core_meta/v3.py [hunk 3]
```diff
@@ -1,5 +1,11 @@
...
+Plattform Industrie 4.0 and does not represent an official document.
+
+WIP Content of Metamodel with Version v3.0.
+Any static content or specific content for this model version can be presented here.
 
 We had to diverge from the book in the following points.
```
Comment (g1zzm0) [inline aas_core_meta/v3.py:10]: Please name the points or remove sentence.

**Review by g1zzm0 — CHANGES_REQUESTED** [review]
Please adapt the docstring

File: aas_core_meta/v3.py (hunk 2 above)
Comment (mristin) [reply aas_core_meta/v3.py:4]: "The presented content is no official ..."

Docstring is going to be translated to SDK code, HTML etc. The term "documentation" might be misleading in many contexts.

File: aas_core_meta/v3.py (hunk 3 above)
Comment (mristin) [reply aas_core_meta/v3.py:10]: I'd remove the sentence. Bear in mind that it makes no sense in a context of an SDK.

File: aas_core_meta/v3.py (hunk 2 above)
Comment (mristin) [reply aas_core_meta/v3.py:4]: Please also remove any mentions of "website" as this is misleading in context of SDKs etc.

Comment (s-heppner) [reply aas_core_meta/v3.py]: See 59c1702a0ef8809115c3eaa8823a0a1dc3435726

File: aas_core_meta/v3.py (hunk 3 above)
Comment (s-heppner) [reply aas_core_meta/v3.py:10]: See S59c1702a0ef8809115c3eaa8823a0a1dc3435726

Comment (mristin) [reply aas_core_meta/v3.py:10]: Thanks, LGTM!

File: aas_core_meta/v3.py (hunk 2 above)
Comment (mristin) [reply aas_core_meta/v3.py]: Thanks, LGTM!

File: aas_core_meta/v3.py (hunk 1 above)
Comment (s-heppner) [reply aas_core_meta/v3.py]: See 59c1702a0ef8809115c3eaa8823a0a1dc3435726

File: aas_core_meta/v3.py [hunk 4]
```diff
@@ -1,5 +1,11 @@
 """
-Provide the meta-model for Asset Administration Shell V3.0.
+Provide an implementation of the Asset Administration Shell (AAS) V3.0.
+
+The presented version of the Metamodel is related to the work of
+aas-core-works, which can be found here: https://github.com/aas-core-works
```
Comment (mristin) [inline aas_core_meta/v3.py:5]:
```suggestion
aas-core-works, which can be found here: https://github.com/aas-core-works.
```

**Pull request merged**